.github/workflows/ci.yml

```diff
@@ -5,11 +5,16 @@ on:
     branches:
       - dev
   pull_request:
+    branches:
+      - dev
+      - master
```

Comment:
@grst  you want to add the skip condition and testing of master with 2 NXF versions as we have in [methylseq](https://github.com/nf-core/methylseq/blob/master/.github/workflows/ci.yml) or fetchngs ?

Reply:
@sateeshperi, added it but it hasn't skipped the checks. What am I missing?

Reply:
ah sorry, got it. This requires adding a test for `skip` in all subsequent steps.

Reply:
@sateeshperi, I came up with an alternative approach that doesn't need this. LMK what you think.

Reply:
cool